client/src/components/ui/itemrow/PersonRowLocal.svelte

```diff
@@ -5,39 +5,45 @@
     import RowTemplate from '../RowTemplate.svelte';
 
     import { PersonPayload, IconSize, Events, RowType } from '../../types.ts';
+    import { allOffices } from '../../../pages/dashboard/stores/OfficeStore.ts';
     import { User } from '../../../../../model/src/user.ts';
     import { Staff } from '../../../../../model/src/staff.ts';
+    import { Office } from '~model/office.ts';
+    
     export let iconSize: IconSize;
     
     // From user.ts
     export let id: User['id'];
     export let name: User['name'];
     export let email: User['email'];
     export let picture: User['picture'];
-    export let globalPermission: User['permission'];
 
     // From staff.ts
     export let office: Staff['office'];
-    export let localPermission: Staff['permission'];
+    export let permission: Staff['permission'];
 
     const dispatch = createEventDispatcher();
     const rowEvent: PersonPayload = {
         ty: RowType.Person,
         id,
         office,
     };
+
+    let officeName: Office['name'];
```

Comment:
No need for this. Svelte [automatically](https://svelte.dev/docs#component-format-script-3-$-marks-a-statement-as-reactive) `let`-declares reactive variables.

```suggestion
```